Source/Types/Script.swift

```diff
@@ -13,14 +13,6 @@ public struct Script: Codable, Param {
     public let args: [HexString]
     public var binaryHash: H256
 
-    public init(version: UInt8, binary: HexString?, reference: H256?, signedArgs: [HexString], args: [HexString]) {
```

Comment:
If delete this Initialization method, we can not initialize `Script` in other  projects.

Reply:
Look at the end of the class...